core/controllers/pages.py

```diff
@@ -18,11 +18,13 @@
 import urlparse
 
 from core.controllers import base
-from core.controllers import editor
 from core.domain import config_domain
 import feconf
 
 
+MODERATOR_REQUEST_FORUM_URL_DEFAULT_VALUE = (
```

Comment:
Why does this still exist? I wasn't aware that it had any uses besides the one we are removing.